sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -976,6 +976,8 @@
   public static final String TASK_COUNT="taskCount";
   public static final String ERROR_VISUALIZATION_THRESHOLD="sunbird_user_upload_error_visualization_threshold";
   public static final String NESTED_KEY_FILTER = "nestedFilters";
+  public static final String IS_STATE_VALIDATED = "isStateValidated";
+  public static final String FLAGS_VALUE_SMALL_CASE = "flagsvalue";
```

Comment:
we can used cassandracolumnname.properties for this

Reply:
Done